newton/_src/utils/render.py

```diff
@@ -20,7 +20,7 @@
 import numpy as np
 import warp as wp
 from warp.render import OpenGLRenderer, UsdRenderer
-from warp.render.utils import solidify_mesh, tab10_color_map
+from warp.render.utils import compute_contact_points, solidify_mesh, tab10_color_map
```

Comment:
This kernel used to be in render.py but it is definitely not part of `warp.render.utils`.
I will add this kernel back into render.py for now.